prow/cmd/deck/static/common/common.ts

```diff
@@ -169,13 +169,16 @@ export namespace tooltip {
 }
 
 export namespace icon {
-  export function create(iconString: string, tip: string = ""): HTMLAnchorElement {
+  export function create(iconString: string, tip: string = "", onClick: string = ""): HTMLAnchorElement {
```

Comment:
This should take a function, not a string:

```suggestion
  export function create(iconString: string, tip: string = "", onClick: (this: HTMLElement, ev: MouseEvent) => any): HTMLAnchorElement {
```

(`this: HTMLElement, ev: MouseEvent) => any` is the type of the click callback)

Reply:
I have it as a string is because I can't get the typescript to recognize/import the `gtag` function. Do you know how to do this?

Reply:
Running `bazel run @nodejs//:bin/yarn -- add @types/gtag.js` should pull in suitable type definitions.